sedai/README.md

```diff
@@ -0,0 +1,47 @@
+# [Sedai][1] — Find out why production is the new playground with a free 14 day trial
```

Comment:
I don't believe level 1 headings are supported either - I'd convert this to a level 3 or just leave it as body text.

Reply:
done